cmd/subnetcmd/join.go

```diff
@@ -133,7 +153,10 @@ but until the node is whitelisted, it will not be able to validate this subnet.`
 		return nil
 	}
 
+	// if **both** flags were set, nothing special needs to be done
```

Comment:
oof, these comments are showing me how confusing this is. We don't need to fix now, but we probably want to think about refactoring in the future to make simpler/more clear.